src/icons/crypto/Blocks.tsx

```diff
@@ -0,0 +1,1832 @@
+import type {SvgIconProps} from '@mui/material';
+import {SvgIcon} from '@mui/material';
+import React from 'react';
+
+const Blocks = React.forwardRef<
+  SVGSVGElement,
+  {
+    iconRef?: React.ForwardedRef<SVGSVGElement>;
+  } & SvgIconProps
```

Comment:
This is a huge icon, can we ask the designer to optimize it for 32x32 px size?

Reply:
Alex created the 36x36 icon for me and I added it to the `crypto/Blocks` path as `Blocks36x36` name and to the storybook as well

Reply:
Also increased the icon containers width, set icon names font size to 14px, and added a tooltip to each icon with both import pathes (default and named) with the Copy buttons. I wasn't sure how to add a font to Storybook and haven't had a chance to figure it out yet.

Reply:
https://unstoppabledomains.github.io/UI-Kit/?path=/story/components-icons--crypto-icons

Reply:
I'd postpone improving font families for later if it's not critical